### ISSUE TITLE
Implement a VPN "lockdown" mode

### ISSUE THREAD

**Issue by bnvk** [CyborgUnplug/CyborgUnplug#21]
Users SHOULD be able to choose an option when they are connected to an CyborgUnplug device that they are unable to have any internet access unless it is tunneled through the VPN.

**Comment by JulianOliver** [comment]
Good idea. Where would it be implemented in the UI? On the 'first boot' setup page an option could be provided. After that, it could be a button that becomes visible after setting up a VPN for the first time, a 'always connect through this vpn' or similar. Thoughts?

**Comment by JulianOliver** [comment]
Just implemented https://github.com/CyborgUnplug/CyborgUnplug/commit/43c41b504f54df251c0239eb244a7b2a210f4a3e for the default Cyborg Unplug VPN. Works well. 

Now implementing for external VPNs

**Comment by JulianOliver** [comment]
OK, Just committed https://github.com/CyborgUnplug/CyborgUnplug/commit/e59728fcc53998c740969f8f73288b3ae0fd6da8. Enables external OpenVPNs (IPredator, etc) to also be made the default route for the plug, persistent over reboots. Tested with VPNBook's Canadian VPN on port 53. 

Will now close this issue.

**Comment by bnvk** [comment]
Based on testing this functionality, connecting works perfectly. The Unplug seems to persist saving the uploaded config file, but does not currently present this to the user as a "Configured Option" that makes easy of connecting again.
